src/SFA.DAS.AdminService.Web/Views/Apply/Applications/Section.cshtml

```diff
@@ -220,7 +220,7 @@
                                         }
                                         else
                                         {
-                                            if (@answer.Value != ",,")
+                                            if (@answer.Value != null && @answer.Value.Any())
                                             {
                                                  <p class="govuk-body">@answer.Value</p>
```

Comment:
This needs to be `@answer.Value[0]` for simple answers but would need to loop through the array for Checkboxlist answers.

Reply:
Thanks. Yes that's what I said!